presto-jdbc/src/main/java/io/prestosql/jdbc/ConnectionProperties.java

```diff
@@ -159,6 +160,15 @@ public ApplicationNamePrefix()
         }
     }
 
+    private static class ClientTags
+            extends AbstractConnectionProperty<String>
+    {
+        public ClientTags()
+        {
+            super("clientTags", NOT_REQUIRED, ALLOWED, STRING_CONVERTER);
```

Comment:
Let's name this `ClientTags` so that it's consistent with the connection client info property.